cfgov/v1/forms.py

```diff
@@ -160,6 +174,37 @@ def clean(self):
             self.data = data
         return self.cleaned_data
 
+    def set_interpreted_date_values(self, cleaned_data):
+        from_date = cleaned_data.get('from_date')
+        to_date = cleaned_data.get('to_date')
+        # if from_ or to_ is filled in, fill them both with sensible values
+        # if neither is filled in, leave them both blank
+        if from_date or to_date:
+            if from_date:
+                self.data['from_date'] = date.strftime(
+                    cleaned_data['from_date'], '%m/%d/%Y')
```

Comment:
Minor suggestion, to avoid duplicating `%m/%d/%Y` a few times: what about adding a property of the form class, e.g.

```py
class FilterableListForm(forms.Form):
   ...
   preferred_datetime_format = '%m/%d/%y'
```

so then later you can refer to it as `self.preferred_datetime_format` and reduce duplication.

Reply:
Good call. Done.